server/lists/adapters/workstacks.js

```diff
@@ -138,10 +144,19 @@ const bindDisplayElements = fromStaticList => async (stage) => {
     }
     stage.deadlineDisplay = formatDate(stage.deadline);
     if (stage.data && stage.data.DueDate) {
-        stage.stageTypeWithDueDateDisplay = stage.stageTypeDisplay + ' due: ' + formatDate(stage.data.DueDate);
+        stage.stageTypeWithDueDateDisplay = stage.stageTypeDisplay + ' due: ' + formatDate(stage.stage.data.DueDate);
```

Comment:
This has heavily changed and we no longer use DueDate as a result of the new MultipleContributions piece.

Reply:
I have now rebased